Tribler/Test/test_my_channel.py

```diff
@@ -125,6 +129,88 @@ def do_page():
 
         self.startTest(do_page)
 
+    def test_metadata(self):
+        def take_screenshot_quit():
+            self.screenshot('Home')
+            self.quit()
+
+        def check_home():
+            self.CallConditional(20, lambda: len(self.guiUtility.frame.home.aw_panel.list.GetItems()) == 1,
+                                 take_screenshot_quit, "No thumbnail torrent in Home panel")
+
+        def on_channel_torrent_created(subject, events, object_id, data_list):
+            add_metadata = False
```

Comment:
We already have a test which uses an rss-feed to create item in a channel. Can't you change the rss stuffs a bit to also accept a local rss.xml file? That should be pretty trivial.

As for the ffmpeg stuffs, not all rss-feeds have valid thumbnails attached to the items. It's a nice fallback to be able to generate thumbnails for files without a thumbnail in the rss/.torrents added from a file.